react/features/google-api/googleApi.js

```diff
@@ -240,8 +242,12 @@ const googleApi = {
                     return null;
                 }
 
+                // user can edit the events, so we want only those that
+                // can be edited
                 return this._getGoogleApiClient()
-                    .client.calendar.calendarList.list();
+                    .client.calendar.calendarList.list({
+                        'minAccessRole': 'writer'
+                    });
```

Comment:
Will this mean user could potentially not see meetings in the list of upcoming events?

Reply:
Yes, hum ... I think I should change that. 
We should query all calendars and show events that has url, but show events without url only for those calendars we can edit. Will fix it.